packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -58,7 +58,6 @@ export const enableFilterEmptyStringAttributesDOM = false;
 export const disableNativeComponentFrames = false;
 export const skipUnmountedBoundaries = false;
 export const deletedTreeCleanUpLevel = 3;
-export const enableSuspenseLayoutEffectSemantics = false;
```

Comment:
This will turn it on in RN then, correct? Probably fine since layout effects are less common there.